stdpopsim/cli.py

```diff
@@ -370,14 +371,21 @@ def run_simulation(args):
         contig = species.get_contig(
             args.chromosome, genetic_map=args.genetic_map,
             length_multiplier=args.length_multiplier)
+        engine = stdpopsim.get_engine(args.engine)
         logger.info(
             f"Running simulation model {model.name} for {species.name} on "
-            f"{contig} with {len(samples)} samples")
-        ts = model.simulate(contig, samples, args.seed)
+            f"{contig} with {len(samples)} samples using {engine.name}.")
+
+        kwargs = vars(args)
+        for var in ("model", "contig", "samples"):
```

Comment:
We don't need to do this now, right?

Reply:
I'm not sure which *exact* thing you're referring to... Did I misunderstand your earlier comment about how to clean up the CLI/kwargs?

In the future, I anticipate that engines (in particular, the SLiM engine) will want their own command line arguments, and this kwargs mechanism is the cleanest way I could think of to get at those CLI arguments from the engine's `simulate()` function. Currently, the `seed` CLI argument is the only parameter in this kwargs dict that is being used. It is named explicitly in the parameter list for `_MsprimeEngine.simulate()`. I guess a seed parameter could instead be part of the API---i.e. have it as a named parameter to the abstract `Engine.simulate()`.

`vars(args)` produces a dictionary of the CLI args, which contains keys for both `model` and `samples` as these names are used in calls to `species_parser.add_arguments()`. So it's necessary to remove these from the dictionary, as the names clash with the explicit model and samples objects we wish to give `simulate()`.

Reply:
The vars thing is very neat --- I just meant we don't need the for loop, because we can call 
```python
engine.simulate(**kwargs)
```
now, right?

Reply:
Yes, good point. I've fixed this, and squashed all the commits, ready for merging.